conans/client/remote_registry.py

```diff
@@ -1,5 +1,6 @@
 import os
```

Comment:
The registry manages references without revision, so all the revisions points to the same remote.

Reply:
**Pending**: Probably there are pending issues not tested removing a revision, is someone removing the entry in the registry even when there are revisions in the local cache?